### PR TITLE
Rework webplatform + farmmanagement

### PR DESCRIPTION
**Pull request by tobiaschielens**

### CODE REVIEW

File: ThreeBotPackages/package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (abom) [inline ThreeBotPackages/package.json]: Why do we have package.json at this top level directory?

File: ThreeBotPackages/yarn.lock [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (abom) [inline ThreeBotPackages/yarn.lock]: The same as https://github.com/threefoldtech/jumpscaleX_threebot/pull/393.
Why do we have yarn.lock here?

**Comment by abom** [comment]
The same comments as https://github.com/threefoldtech/jumpscaleX_threebot/pull/393, also, it contains nearly the same changes, should be reviewed again after merging #393 or review #393 after merging this! but still, too much changes, better to be done in a smaller PRs for every change type.

**Comment by tobiaschielens** [comment]
I did remove the package.json and yarn.lock

**Pull request merged**